components/raftstore/src/store/worker/split_config.rs

```diff
@@ -19,6 +19,16 @@ const DEFAULT_SPLIT_BALANCE_SCORE: f64 = 0.25;
 // We get contained score by sample.contained/(sample.right+sample.left+sample.contained). It will be used to avoid to split regions requested by range.
 const DEFAULT_SPLIT_CONTAINED_SCORE: f64 = 0.5;
 
+// When the gRPC poll thread CPU usage is higher than gRPC poll thread count * `DEFAULT_GRPC_THREAD_CPU_OVERLOAD_THRESHOLD_RATIO`,
+// the CPU-based split won't be triggered to prevent from increasing the CPU usage.
+const DEFAULT_GRPC_THREAD_CPU_OVERLOAD_THRESHOLD_RATIO: f64 = 0.5;
+// When the Unified Read Poll thread CPU usage is higher than Unified Read Poll thread count * `DEFAULT_UNIFIED_READ_POOL_THREAD_CPU_OVERLOAD_THRESHOLD_RATIO`,
+// the CPU-based split will be triggered to split the top hot CPU region.
```

Comment:
I suggest explaining the priority of these parameters.

Reply:
What do you mean by "priority"? I think these thresholds don't quite have a term on priority.

Reply:
I mean the case when `DEFAULT_GRPC_THREAD_CPU_OVERLOAD_THRESHOLD_RATIO` is more than 0.5 and `DEFAULT_UNIFIED_READ_POOL_THREAD_CPU_OVERLOAD_THRESHOLD_RATIO` is more than 0.8.